docs: add bi-weekly-report (0115-0128).md (#1764)
iresty/iresty.github.io: blog/en/blog/2024/01/31/bi-weekly-report.md

---
title: "Biweekly Report (January 15 - January 28)"
keywords: ["Apache APISIX", "API Gateway", "Weekly Report", "Contributor"]
description: Our bi-weekly Apache APISIX community report is your window into the project's weekly developments. It is a tool to facilitate your seamless integration into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
tags: [Community]
image: https://static.apiseven.com/uploads/2024/01/30/CeII2ZiX_20240130_EG_Cover.png
---

> > We have recently made some additions and improvements to specific features within Apache APISIX. The update is adding the `ocsp-stapling` plugin. For additional information, please consult the bi-weekly report.

Reviewer: ```suggestion
> We have recently added the `ocsp-stapling` plugin within Apache APISIX. Please read the bi-weekly report for more details.
```

<!--truncate-->
## Introduction

From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'Many hands make light work' rings true in our way, made possible by the collective dedication of our community.

From 01.15 to 01.28, a total of 19 contributors made 38 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.

> We have recently added and enhanced several plugins, and here is a summary of the update:

Reviewer: ```suggestion
We have recently added a plugin:
```
